testsuite/features/step_definitions/common_steps.rb

```diff
@@ -98,27 +98,26 @@
 
   # the moment.js library being used has some weird rules, which these conditionals follow
   if (uptime[:days] >= 1 && rounded_uptime_days < 2) || (uptime[:days] < 1 && rounded_uptime_hours >= 22) # shows "a day ago" after 22 hours and before it's been 1.5 days
-    step %(I should see a "a day ago" text)
+    step 'I should see a "a day ago" text'
   elsif rounded_uptime_hours > 1 && rounded_uptime_hours <= 21
     step %(I should see a "#{rounded_uptime_hours} hours ago" text)
   elsif rounded_uptime_minutes >= 45 && rounded_uptime_hours == 1 # shows "an hour ago" from 45 minutes onwards up to 1.5 hours
-    step %(I should see a "an hour ago" text)
+    step 'I should see a "an hour ago" text'
   elsif rounded_uptime_minutes > 1 && rounded_uptime_hours <= 1
     step %(I should see a "#{rounded_uptime_minutes} minutes ago" text)
   elsif uptime[:seconds] >= 45 && rounded_uptime_minutes == 1
-    step %(I should see a "a minute ago" text)
+    step 'I should see a "a minute ago" text'
   elsif uptime[:seconds] < 45
-    step %(I should see a "a few seconds ago" text)
+    step 'I should see a "a few seconds ago" text'
   elsif rounded_uptime_days < 25
     step %(I should see a "#{rounded_uptime_days} days ago" text) # shows "a month ago" from 25 days onwards
   else
-    step %(I should see a "a month ago" text)
+    step 'I should see a "a month ago" text'
   end
 end
 
-Then(/^I should see several text fields for "([^"]*)"$/) do |host|
-  node = get_target(host)
-  steps %(Then I should see a "UUID" text
+Then(/^I should see several text fields$/) do
+  steps 'Then I should see a "UUID" text
```

Comment:
I think this is also wrong, the grouping of steps needs a specific syntax with parenthesis

Reply:
https://github.com/uyuni-project/uyuni/pull/7444#discussion_r1300452738